pkg/actuators/machine/machine.go

```diff
@@ -249,12 +249,6 @@ func (ms *machineScope) delete() error {
 	if err != nil {
```

Comment:
It seems we should check for the ENotFound error code here. Use the ovirtclient.HasErrorCode function.